reana_server/reana_admin/main.py

```diff
@@ -639,3 +640,61 @@ def set_default_quota_limit(ctx):
             resource_type=resource.type_.name,
             limit=DEFAULT_QUOTA_LIMITS.get(resource.type_.name),
         )
+
+
+@reana_admin.command("consume-queue")
+@click.option(
+    "--queue-name",
+    "-q",
+    required=True,
+    type=str,
+    help="Name of the queue that will be consumed, e.g workflow-submissions",
+)
+@click.option(
+    "--key",
+    "-k",
+    type=str,
+    help="Key that will be used to get the id value from the message body, e.g workflow_name_or_id",
+)
+@click.option(
+    "--messages-to-delete",
+    "-m",
+    multiple=True,
+    help="List of message ids to remove from the queue, e.g UUID of a workflow",
+)
```

Comment:
I think it is better if we refer to these as `values` instead of `message ids`. WDYT?

Reply:
Yes. It is more general and better describes.

Should we still keep `--messages-to-delete`? Maybe we should use `--values-to-delete`? WDYT?

Reply:
I agree that `--values-to-delete` is better. It should be changed also in the rest of the code.